.eslintrc.js

```diff
@@ -15,16 +15,53 @@
  */
 module.exports = {
   'root': true,
+  'env': {
+    'node': true,
+  },
+  'plugins': ['header'],
```

Comment:
As I can see you can bring here `prettier` too, because it is used in both the `'**/*.ts'` and the `'**/*.js` overrides.

Reply:
See response here: https://github.com/IBM/cloudant-node-sdk/pull/545#discussion_r751220149
> prettier formats the code differently, if I import it on project level. So I would leave currently as-it-is now.

Reply:
Do you mean this way: 280d701 ?

Reply:
Yes, exactly!

Reply:
If I make the fill replace, then in local environment I receive:
```
1215 problems (1215 errors, 0 warnings)
  1192 errors and 0 warnings potentially fixable with the `--fix` option.
```
Let see what will happen in the CI env: a00953a

Reply:
🤞

Reply:
The same amount of erros:
```
[2021-11-19T14:50:08.605Z] ��� 1215 problems (1215 errors, 0 warnings)
[2021-11-19T14:50:08.605Z]   1192 errors and 0 warnings potentially fixable with the `--fix` option.
```
https://cloudant-sdks-jenkins.swg-devops.com/blue/organizations/jenkins/SDKs%2Fcloudant-node-sdk/detail/update-eslint/13/pipeline

Reply:
After auto-fix there are the following problems
```
281 problems (281 errors, 0 warnings)
258 errors and 0 warnings potentially fixable with the `--fix` option.
```
They are recursive formatting bugs. 😆

Reply:
I think this direction not really worth to go to. I prefer the state before the refactor.

Reply:
Okay, I see, it has extreme problems, interesting that they do not occur in the "before" case, but I agree this is out of scope. Just leave as it was previously!

Reply:
Thanks for checking!
I removed 280d701 and a00953a.